Edit Inference Endpoint

Samiul-TheSoccerFan opened the pull request:
## Summary

This PR introduces edit functionality for inference endpoints in the inference management UI. To support this, the following changes were made:

- Refactored the `kbn-inference-endpoint-ui-common` package to include support for edit mode.
- Moved the add inference endpoint functionality into the common plugin.
- Updated each plugin to handle opening and closing the flyout, as well as providing a method for their specific task.
- Refactored the way actions are defined in the inference management UI.


https://github.com/user-attachments/assets/14d69f18-dc6e-4521-89fd-9c21e9b2443c



### Checklist

Check the PR satisfies following conditions. 

Reviewers should verify this PR satisfies this list as well.

- [X] Any text added follows [EUI's writing guidelines](https://elastic.github.io/eui/#/guidelines/writing), uses sentence case text and includes [i18n support](https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-i18n/README.md)
- [X] [Unit or functional tests](https://www.elastic.co/guide/en/kibana/master/development-tests.html) were updated or added to match the most common scenarios
- [X] This was checked for breaking HTTP API changes, and any breaking changes have been approved by the breaking-change committee. The `release_note:breaking` label should be applied in these situations.
- [ ] [Flaky Test Runner](https://ci-stats.kibana.dev/trigger_flaky_test_runner/1) was used on any tests changed
- [ ] The PR  description includes the appropriate Release Notes section, and the correct `release_note:*` label is applied per the [guidelines](https://www.elastic.co/guide/en/kibana/master/contributing.html#kibana-release-notes-process)





<!--ONMERGE {"backportTargets":["8.x"]} ONMERGE-->

elasticmachine commented:
<!-- buildkite-pr-bot-trigger-comment -->:robot: Jobs for this PR can be triggered through checkboxes. :construction: 

:information_source: To trigger the CI, please tick the checkbox below :point_down: 

- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-pull-request --> Click to trigger **kibana-pull-request** for this PR!
- [ ] <!-- buildkite-pr-bot-checkbox-trigger:kibana-deploy-project-from-pr --> Click to trigger **kibana-deploy-project-from-pr** for this PR!

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/270752)
* Commit: 985b9d12454ed184ff05bb486fd9f5fa4f170c2e

### Failed CI Steps
* [Quick Checks](https://buildkite.com/elastic/kibana-pull-request/builds/270752#0194aa28-3551-4587-a9b5-ba9df3f14b06)

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270752","number":270752,"commit":"985b9d12454ed184ff05bb486fd9f5fa4f170c2e"}],"number":270752}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/270778)
* Commit: 86ad6164b9e76e3f1e791353fa80d64db00b1aaa

### Failed CI Steps
* [Jest Tests #3](https://buildkite.com/elastic/kibana-pull-request/builds/270778#0194aa86-13d8-47b4-946a-ad2e9dd96227)
* [Jest Tests #3](https://buildkite.com/elastic/kibana-pull-request/builds/270778#0194aaa7-d004-4d0a-9940-d15a03361a75)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/270778#0194aa86-13d8-47b4-946a-ad2e9dd96227) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/270778/jobs/0194aa86-13d8-47b4-946a-ad2e9dd96227/artifacts/0194aaa7-bd5b-41fe-8630-074f356c1503) Jest Tests #<span></span>3 / InferenceFlyout handles edit mode correctly
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/270778#0194aaa7-d004-4d0a-9940-d15a03361a75) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/270778/jobs/0194aaa7-d004-4d0a-9940-d15a03361a75/artifacts/0194aaac-cf17-4a14-be77-bdeee34d42df) Jest Tests #<span></span>3 / InferenceFlyout handles edit mode correctly

### History
* :broken_heart: [Build #270752](https://buildkite.com/elastic/kibana-pull-request/builds/270752) failed 985b9d12454ed184ff05bb486fd9f5fa4f170c2e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270778","number":270778,"commit":"86ad6164b9e76e3f1e791353fa80d64db00b1aaa"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270752","number":270752,"commit":"985b9d12454ed184ff05bb486fd9f5fa4f170c2e"}],"number":270778}
buildkite-pr-comment-->

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress, with failures
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/270787)
* Commit: 4447a3ac7f88278b26823d1ecdcce5c3112f5307

### Failed CI Steps
* [Jest Tests #5](https://buildkite.com/elastic/kibana-pull-request/builds/270787#0194aae3-9276-4a98-a0b4-30d9fa42f502)
* [Jest Tests #5](https://buildkite.com/elastic/kibana-pull-request/builds/270787#0194ab11-2ae8-49f2-9456-6e0149dbccf5)
* [Jest Tests #9](https://buildkite.com/elastic/kibana-pull-request/builds/270787#0194aae3-9277-4ec6-925e-b58eab50543f)

### Test Failures
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/270787#0194aae3-9277-4ec6-925e-b58eab50543f) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/270787/jobs/0194aae3-9277-4ec6-925e-b58eab50543f/artifacts/0194ab14-ec10-4555-85a7-cab2692ec57e) Jest Tests #<span></span>9 / HoverableAvatar renders the tooltip when hovering
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/270787#0194aae3-9276-4a98-a0b4-30d9fa42f502) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/270787/jobs/0194aae3-9276-4a98-a0b4-30d9fa42f502/artifacts/0194ab10-e476-41f9-ab9d-82ef41d71470) Jest Tests #<span></span>5 / InferenceFlyout handles edit mode correctly
* [[job]](https://buildkite.com/elastic/kibana-pull-request/builds/270787#0194ab11-2ae8-49f2-9456-6e0149dbccf5) [[logs]](https://buildkite.com/organizations/elastic/pipelines/kibana-pull-request/builds/270787/jobs/0194ab11-2ae8-49f2-9456-6e0149dbccf5/artifacts/0194ab15-c504-4639-a877-5c487cfca5eb) Jest Tests #<span></span>5 / InferenceFlyout handles edit mode correctly

### History
* :broken_heart: [Build #270778](https://buildkite.com/elastic/kibana-pull-request/builds/270778) failed 86ad6164b9e76e3f1e791353fa80d64db00b1aaa
* :broken_heart: [Build #270752](https://buildkite.com/elastic/kibana-pull-request/builds/270752) failed 985b9d12454ed184ff05bb486fd9f5fa4f170c2e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"failing","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270787","number":270787,"commit":"4447a3ac7f88278b26823d1ecdcce5c3112f5307"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270778","number":270778,"commit":"86ad6164b9e76e3f1e791353fa80d64db00b1aaa"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270752","number":270752,"commit":"985b9d12454ed184ff05bb486fd9f5fa4f170c2e"}],"number":270787}
buildkite-pr-comment-->

leemthompo commented on x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/translations.ts:
```diff
@@ -149,3 +149,31 @@ export const CANCEL = i18n.translate('xpack.inferenceEndpointUICommon.components
+export const ENDPOINT_CREATION_FAILED = i18n.translate(
+  'xpack.inferenceEndpointUICommon.components.endpointAddedFailure',
+  {
+    defaultMessage: 'Endpoint creation failed',
```
```suggestion
    defaultMessage: 'Inference endpoint creation failed',
```

leemthompo commented on x-pack/platform/packages/shared/kbn-inference-endpoint-ui-common/src/translations.ts:
```diff
@@ -149,3 +149,31 @@ export const CANCEL = i18n.translate('xpack.inferenceEndpointUICommon.components
+export const ENDPOINT_UPDATE_FAILED = i18n.translate(
+  'xpack.inferenceEndpointUICommon.components.endpointUpdateFailure',
+  {
+    defaultMessage: 'Endpoint update failed',
```
```suggestion
    defaultMessage: 'Inference endpoint update failed',
```

leemthompo reviewed (commented):
Just a little copy nit for consistency

darnautov commented:
During my local testing I noticed the endpoint update hangs with timeout quite often

<img width="689" alt="image" src="https://github.com/user-attachments/assets/a5fee944-ea8c-47f9-a07f-e6ce07f9d25b" />

```
[2025-01-28T15:11:34.583+01:00][ERROR][plugins.inferenceEndpoint] TimeoutError: Request timed out
    at KibanaTransport._request (/Users/darnautov/repos/kibana/node_modules/@elastic/transport/lib/Transport.js:574:50)
    at async /Users/darnautov/repos/kibana/node_modules/@elastic/transport/lib/Transport.js:641:32
    at async KibanaTransport.request (/Users/darnautov/repos/kibana/node_modules/@elastic/transport/lib/Transport.js:637:20)
    at /Users/darnautov/repos/kibana/x-pack/platform/plugins/shared/inference_endpoint/server/routes/index.ts:118:77
    at addInferenceEndpoint (/Users/darnautov/repos/kibana/x-pack/platform/plugins/shared/inference_endpoint/server/lib/add_inference_endpoint.ts:15:30)
    at /Users/darnautov/repos/kibana/x-pack/platform/plugins/shared/inference_endpoint/server/routes/index.ts:118:77
    at async CoreVersionedRoute.handle (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/versioned_router/core_versioned_route.ts:104:22)
    at async Router.handle (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/router.ts:127:30)
    at async handler (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/router.ts:113:52)
    at async exports.Manager.execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/toolkit.js:60:28)
    at async internals.handler (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/handler.js:46:20)
    at async exports.execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/handler.js:31:20)
    at async Request._lifecycle (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:370:32)
    at async Request._execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:280:9)
    at /Users/darnautov/repos/kibana/x-pack/platform/plugins/shared/inference_endpoint/server/routes/index.ts:113:38
    at Inference.delete (/Users/darnautov/repos/kibana/node_modules/@elastic/elasticsearch/lib/api/api/inference.js:31:17)
    at /Users/darnautov/repos/kibana/x-pack/platform/plugins/shared/inference_endpoint/server/routes/index.ts:113:38
    at async CoreVersionedRoute.handle (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/versioned_router/core_versioned_route.ts:104:22)
    at async Router.handle (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/router.ts:127:30)
    at async handler (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/router.ts:113:52)
    at async exports.Manager.execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/toolkit.js:60:28)
    at async internals.handler (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/handler.js:46:20)
    at async exports.execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/handler.js:31:20)
    at async Request._lifecycle (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:370:32)
    at async Request._execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:280:9)
    at /Users/darnautov/repos/kibana/x-pack/platform/plugins/shared/inference_endpoint/server/routes/index.ts:108:39
    at Object.get (/Users/darnautov/repos/kibana/src/core/packages/http/context-server-internal/src/context_container.ts:83:14)
    at /Users/darnautov/repos/kibana/x-pack/platform/plugins/shared/inference_endpoint/server/routes/index.ts:108:39
    at Object.fn (/Users/darnautov/repos/kibana/src/core/packages/http/context-server-internal/src/context_container.ts:56:16)
    at CoreVersionedRoute.handle (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/versioned_router/core_versioned_route.ts:104:36)
    at handler (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/versioned_router/core_versioned_route.ts:170:32)
    at Router.handle (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/router.ts:127:36)
    at handler (/Users/darnautov/repos/kibana/src/core/packages/http/router-server-internal/src/router.ts:113:63)
    at exports.Manager.execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/toolkit.js:57:29)
    at internals.handler (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/handler.js:46:48)
    at exports.execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/handler.js:31:36)
    at Request._lifecycle (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:370:68)
    at async Request._execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:280:9)
    at Request._lifecycle (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:370:86)
    at Request._invoke (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:392:18)
    at Request._lifecycle (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:370:86)
    at async Request._execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:280:9)
    at Request._lifecycle (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:370:86)
    at internals.payload (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/route.js:403:36)
    at Request._lifecycle (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:370:68)
    at async Request._execute (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:280:9)
    at Request._lifecycle (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:370:86)
    at module.exports.internals.Auth._authenticate (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/auth.js:252:24)
    at authenticate (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/auth.js:249:21)
    at Request._lifecycle (/Users/darnautov/repos/kibana/node_modules/@hapi/hapi/lib/request.js:370:68)
[2025-01-28T15:11:34.586+01:00][ERROR][http] 500 Server Error
```

Samiul-TheSoccerFan commented:
Hi @darnautov, Thank you adding these two errors.

The first one `TimeoutError: Request timed out` is more likely a resource issue(maybe?). I was able to create an inference endpoint and then was able to increase the thread to `4` without any issues. I have added a tag to deploy in QA so we can test further.

For the second one, as we do not have the `_update` API working, I have added a workaround and now the inference will be created even if it is deleted in the first round. I hope that reduces the flakiness to a certain level.

Please let me know if you have any further questions.

Samiul-TheSoccerFan commented on x-pack/solutions/search/plugins/search_inference_endpoints/public/utils/flatten_object.ts:
```diff
@@ -0,0 +1,21 @@
+ * 2.0.
+ */
+
+export const flattenObject = (obj: Record<string, any>, prefix = ''): Record<string, any> => {
```
Good catch, deleted now

Samiul-TheSoccerFan commented:
> `TimeoutError: Request timed out` is more likely a resource issue(maybe?). I was able to create an inference endpoint and then was able to increase the thread to `4` without any issues. I have added a tag to deploy in QA so we can test further.

TimeoutError: Request timed out is more likely a resource issue(maybe?). I was able to create an inference endpoint and then was able to increase the thread to 4 without any issues. I have added a tag to deploy in QA so we can test further.

Samiul-TheSoccerFan commented:
> There is an issue with the update logic. If set a different Model ID, API returns an error, and original inference endpoints is deleted.
> 
>  Screen.Recording.2025-01-28.at.15.17.47.mov

We do not have the _update API working, I have added a workaround and now the inference will be created even if it is deleted in the first round. I hope that reduces the flakiness to a certain level.
commit: https://github.com/elastic/kibana/pull/207953/commits/f0711eabac356f107660714794f01c8bfa3eed71

efegurkan commented on x-pack/solutions/search/plugins/search_inference_endpoints/public/components/all_inference_endpoints/render_table_columns/render_actions/actions/delete/delete_action.tsx:
```diff
@@ -5,22 +5,23 @@
 interface DeleteActionProps {
   selectedEndpoint: InferenceEndpointUI;
+  onCancel: () => void;
+  displayModal: boolean;
```
nitpick - non-blocking: for some reason it doesn't feel right to get the visibility of this modal from outside.

efegurkan reviewed (approved):
LGTM in general.

darnautov reviewed (approved):
LGTM

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/271052)
* Commit: d8fb6eb966a8b7b6e97ad06f964b698a759fd4fe
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-207953-d8fb6eb966a8`

### History
* :broken_heart: [Build #270787](https://buildkite.com/elastic/kibana-pull-request/builds/270787) failed 4447a3ac7f88278b26823d1ecdcce5c3112f5307
* :broken_heart: [Build #270778](https://buildkite.com/elastic/kibana-pull-request/builds/270778) failed 86ad6164b9e76e3f1e791353fa80d64db00b1aaa
* :broken_heart: [Build #270752](https://buildkite.com/elastic/kibana-pull-request/builds/270752) failed 985b9d12454ed184ff05bb486fd9f5fa4f170c2e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/271052","number":271052,"commit":"d8fb6eb966a8b7b6e97ad06f964b698a759fd4fe"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270787","number":270787,"commit":"4447a3ac7f88278b26823d1ecdcce5c3112f5307"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270778","number":270778,"commit":"86ad6164b9e76e3f1e791353fa80d64db00b1aaa"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270752","number":270752,"commit":"985b9d12454ed184ff05bb486fd9f5fa4f170c2e"}],"number":271052}
buildkite-pr-comment-->

mattkime reviewed (approved):
Codeowner changes look good, Opened and closed the flyout without issue from index management. I did not create an endpoint as I don't know how to do that.

elasticmachine commented:
## :hourglass_flowing_sand: Build in-progress
* [Buildkite Build](https://buildkite.com/elastic/kibana-pull-request/builds/271312)
* Commit: fc71fcea1a7d94ebfa68db3bb99f4dc0fd58dd62
* Kibana Serverless Image: `docker.elastic.co/kibana-ci/kibana-serverless:pr-207953-fc71fcea1a7d`

### History
* :green_heart: [Build #271052](https://buildkite.com/elastic/kibana-pull-request/builds/271052) succeeded d8fb6eb966a8b7b6e97ad06f964b698a759fd4fe
* :broken_heart: [Build #270787](https://buildkite.com/elastic/kibana-pull-request/builds/270787) failed 4447a3ac7f88278b26823d1ecdcce5c3112f5307
* :broken_heart: [Build #270778](https://buildkite.com/elastic/kibana-pull-request/builds/270778) failed 86ad6164b9e76e3f1e791353fa80d64db00b1aaa
* :broken_heart: [Build #270752](https://buildkite.com/elastic/kibana-pull-request/builds/270752) failed 985b9d12454ed184ff05bb486fd9f5fa4f170c2e

<!--buildkite-pr-comment-kibana-pull-request
{"builds":[{"buildStatus":{"state":"running","success":true,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/271312","number":271312,"commit":"fc71fcea1a7d94ebfa68db3bb99f4dc0fd58dd62"},{"buildStatus":{"state":"passed","success":true,"hasRetries":true,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/271052","number":271052,"commit":"d8fb6eb966a8b7b6e97ad06f964b698a759fd4fe"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270787","number":270787,"commit":"4447a3ac7f88278b26823d1ecdcce5c3112f5307"},{"buildStatus":{"state":"failed","success":false,"hasRetries":true,"hasNonPreemptionRetries":true},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270778","number":270778,"commit":"86ad6164b9e76e3f1e791353fa80d64db00b1aaa"},{"buildStatus":{"state":"failing","success":false,"hasRetries":false,"hasNonPreemptionRetries":false},"url":"https://buildkite.com/elastic/kibana-pull-request/builds/270752","number":270752,"commit":"985b9d12454ed184ff05bb486fd9f5fa4f170c2e"}],"number":271312}
buildkite-pr-comment-->

kibanamachine commented:
Starting backport for target branches: 8.x

https://github.com/elastic/kibana/actions/runs/13023868177

kibanamachine commented:
## 💔 All backports failed

| Status | Branch | Result |
|:------:|:------:|:------|
|❌|8.x|Backport failed because of merge conflicts|

### Manual backport
To create the backport manually run:
```
node scripts/backport --pr 207953
```

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sqren/backport)

<!--- Backport version: 9.4.3 -->

Samiul-TheSoccerFan commented:
## 💚 All backports created successfully

| Status | Branch | Result |
|:------:|:------:|:------|
|✅|8.x|[<img src="https://img.shields.io/github/pulls/detail/state/elastic/kibana/208669">](https://github.com/elastic/kibana/pull/208669)|

Note: Successful backport PRs will be merged automatically after passing CI.

### Questions ?
Please refer to the [Backport tool documentation](https://github.com/sorenlouv/backport)

<!--- Backport version: 9.6.4 -->
